modules/amxBidAdapter.js

```diff
@@ -302,8 +303,13 @@ export const spec = {
           bidRequestsCount: 0,
         };
 
+    const transactionId = deepAccess(
+      bidderRequest,
+      'ortb2.ext.tid',
+      deepAccess(bidRequests[0], 'ortb2Imp.ext.tid', generateUUID()));
+
     const payload = {
-      a: bidderRequest.auctionId,
+      a: transactionId,
```

Comment:
Quick question @nickjacob ; do you plan to put this field in ortb2 bid requests to downstream parties ion either source.tid or imp.ext.tid? If so, imp.ext.tid should not fall back to a value that isn't shared by all bidders. The openrtb spec doesnt' allow for exchanges to make up their own value here. Perhaps you would want to pass yourself an indicator of if this value is shared or null if you can't take the null here?